src/applications/mhv/secure-messaging/tests/e2e/draft-page-test/secure-messaging-multiple-drafts-interface.cypress.spec.js

```diff
@@ -35,7 +35,7 @@ describe('handle multiple drafts in one thread', () => {
       .and('contain.text', `${draftsCount} drafts`);
 
     cy.get('[data-testid="reply-form"]')
-      .find('h3')
+      .find('[data-testid="draft-header"]')
```

Comment:
@oleksii-morgun-va I needed to add a test ID to the `h3` because there is another h3 in the wrapping element for screenreaders that was making this test fail.